test/functional/apps/visualize/index.ts

```diff
@@ -81,10 +78,7 @@ export default function ({ getService, loadTestFile }: FtrProviderContext) {
     describe('visualize ciGroup4', function () {
       this.tags('ciGroup4');
 
-      loadTestFile(require.resolve('./_line_chart_split_series'));
-      loadTestFile(require.resolve('./_line_chart_split_chart'));
       loadTestFile(require.resolve('./_pie_chart'));
-      loadTestFile(require.resolve('./_point_series_options'));
       loadTestFile(require.resolve('./_markdown_vis'));
```

Comment:
I would say keep these here and remove the `xy` unneeded test files from here...

https://github.com/elastic/kibana/blob/6b1ca87a4b71c607092315c16c4fcb6dff2150db/test/functional/apps/visualize/index.ts#L45-L53

Then once vislib is gone for good, only the top test block needs to be removed.

Reply:
Nick I def get your point but I will keep them in this group for now. It seems that like that the tests are more balanced between the ci groups. I was thinking to re-organize everything after the vislib removal